<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Продукты</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Белковая пища</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="proteins_cb1">
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Говядина</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Свинина</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Курица</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Яйца</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Печень</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="proteins_cb2">
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Говядина</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Свинина</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Курица</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Яйца</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Печень</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Молочная пища</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="milk_cb1">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Молоко</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Творог</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Простокваша</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="milk_cb2">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Молоко</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Творог</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Простокваша</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Овощи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="vegetables_cb1">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Морковь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Капуста</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зелень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Свекла</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="vegetables_cb2">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Морковь</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Капуста</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зелень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Свекла</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Крупы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="porige_cb">
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Овсянка</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Гречка</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Перловка</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ok_btn">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_btn">
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
